docs/i18n/translating.md

```diff
@@ -0,0 +1,72 @@
+# Translation Guidelines
+To make sure translations are up to par with the standards we hold on the MCprep repo, all translators must follow these guidelines.
+1. No AI for translations, such as Google Translate, ChatGPT, etc. AI translations are unreliable, differ massively in terms of quality between different languages, and don't take context properly into account. 
+2. All translations must be confirmed by a regular user who knows the language in question. This is so we can verify that the translation is up to par.
+
+Failure to follow these guidelines may prevent one from being able to contribute translations for MCprep in the future.
```

Comment:
I think we can remove this line, as it sets a bit of a negative tone. As maintainers, we'll always have the right to refuse contributions when it comes down to it. If we do face it, let's approach it from a teach and learn perspective.